pkg/collector/corechecks/system/disk.go

```diff
@@ -0,0 +1,194 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
```

Comment:
This probably won't compile on windows now that it does not contain any implementation of the OS specific methods. Let's add a `// +build !windows`

Reply:
Oh yes right